bots/psuteam7bot/castle.js

```diff
@@ -95,10 +112,12 @@ castle.buildFromQueue = (self) => {
 
 castle.recordPosition = (self) => {
     let turn = self.me.turn;
-    if(turn == 1){
+    if(turn === 1|| turn === 2){
+        //self.log("Sent x-coord: " + self.me.x);
```

Comment:
Why are we spacing this out to two consecutive turns?

Reply:
So castle 1's castle_talk doesn't expire before castle 2 or castle 3 can read it on the next round.
Same things for castle 2's castle_talk for castle 3.
Another way is to make castles read for 2 turns instead of castle-talking same coords for 2 turns.